src/dev-mock-service/redis-mock-service/index.ts

```diff
@@ -0,0 +1,225 @@
+import type { RedisClient } from "@devvit/public-api";
+import type {
+  BasicRedisMethods,
+  NonKeyRedisParams,
+  RedisOverride,
+  RedisSyncResponse,
+  Unpacked,
+} from "./types.js";
+import type { HandlerOverride } from "../types/internal.js";
+
+export const createDevvRedis = (
```

Comment:
I am not sure about the naming here, because it's not a mockRedis, it's an instance that intercepts some redis requests, but fires real ones for the rest
So calling it `mockRedis` might lead to an expectation that it only performs mock requests

Devv was a way to do the reference to devvit and developer at the same time, and I did not come up with a better option yet